src/main/java/io/reactivesw/category/application/controller/CategoryController.java

```diff
@@ -122,9 +124,10 @@ public CategoryView updateCategory(@PathVariable(value = Router.CATEGORY_ID) Str
    */
   @GetMapping(Router.CATEGORY_WITH_ID)
   public CategoryView getCategoryById(@PathVariable(value = Router.CATEGORY_ID) String id) {
-    LOG.debug("Enter getCategoryById, id is {}.", id);
+    LOG.info("Enter. Get category by id, id is {}.", id);
     CategoryView category = categoryService.getCategoryById(id);
-    LOG.debug("End getCategoryById, get category: {}.", category.toString());
+    LOG.info("Exit. Got category by Id, got category id is {}.", category.getId());
+    LOG.trace("Exit. Got category by Id, got category is {}.", category);
```

Comment:
don't need "Exit" here.
remove "Got category by id,".
use "Category: {}." is ok.